modal/cli/import_refs.py

```diff
@@ -79,23 +91,37 @@ def import_file_or_module(file_or_module: str):
     return module
 
 
-def get_by_object_path(obj: Any, obj_path: str) -> Optional[Any]:
+@dataclass
+class MethodReference:
```

Comment:
This helps with deferring method reference until after the class gets instantiated by the CLI